src/main/ts/utils/deepmap.ts

```diff
@@ -20,9 +20,15 @@ export function deepMap(
         return
       }
 
+      const value = descriptor.get ? descriptor.get() : descriptor.value
+      // descriptor cannot contain both accessor method and value
+      // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#description
+      delete descriptor.get
+      delete descriptor.set
+
       Object.defineProperty(n, key, {
         ...descriptor,
```

Comment:
We should not mutate the original descriptor value. extract the rest of props instead.